### ISSUE TITLE
Show / Hide are shown wrongly

### ISSUE THREAD

**Issue by moshpirit** [rdoeffinger/Dictionary#22]
When we want to download any dictionary, all the available dictionaries are shown and there's a button to Show and Hide them but seems like they appear when they don't have to:

If the dictionaries are shown there's a Show button instead of Hide and vice versa.

**Comment by rdoeffinger** [comment]
I think that's actually supposed to be a kind of "status toggle", not a button.
I.e. it is supposed to show the current setting (aka "works as designed").
As I'm not really convinced it is better the other way round I so far decided to be lazy and leave it as-is, even though I see your point.

**Comment by moshpirit** [comment]
OK, but then it should be Shown/Hidden, right?

**Comment by antonv6** [comment]
I agree that this is an issue, because "Show" and "Hide" are not states, they describe actions the app should do when the button is pressed, and they are reversed right now.

**Comment by Smojo** [comment]
Fully agree. Never really looked at it and I don't use QuickDic that often, but just updated to version 5.6.0 and it was the first thing I was wondering about. I should be more understandable "toogle" thing like a switch or so. 

Otherwise - with a button - it does not make sense. If the list is shown the button should be named as "Hide" if the list is hidden the button should be named "Show".
